src/internet_identity/src/anchor_management/registration.rs

```diff
@@ -122,32 +124,55 @@ fn create_captcha<T: RngCore>(rng: T) -> (Base64, String) {
 }
 
 lazy_static! {
-    /// Map of problematic characters that are easily mixed up by humans to the "normalized" replacement.
-    /// I.e. the captcha will never contain any of the characters in the key set and any input provided
-    /// will be mapped to the matching value.
+    /// Problematic characters that are easily mixed up by humans to the "normalized" replacement.
```

Comment:
nit: this line doesn't seem to be grammatically correct anymore (it was "map of ... chars ... to the ...", now it is only "chars  ... to the ...".